### PR TITLE
[Docs] update docs (Stable unCLIP) to reflect the updated ckpts.

### PR DESCRIPTION
**Pull request by sayakpaul**

### CODE REVIEW

File: docs/source/en/api/pipelines/stable_unclip.mdx [hunk 1]
```diff
@@ -54,8 +50,8 @@ from io import BytesIO
 from diffusers import StableUnCLIPImg2ImgPipeline
 
 pipe = StableUnCLIPImg2ImgPipeline.from_pretrained(
-    "fusing/stable-unclip-2-1-l-img2img", torch_dtype=torch.float16
-)  # TODO update model path
+    "stabilityai/stable-diffusion-2-1-unclip", torch_dtype=torch.float16
```
Comment (patrickvonplaten) [inline docs/source/en/api/pipelines/stable_unclip.mdx:53]:
```suggestion
    "stabilityai/stable-diffusion-2-1-unclip", torch_dtype=torch.float16, variation="fp16"
```

File: docs/source/en/api/pipelines/stable_unclip.mdx [hunk 2]
```diff
@@ -64,6 +60,13 @@ response = requests.get(url)
...
+images = pipe(init_image).images
+images[0].save("fantasy_landscape.png")
+```
+
+Optionally, you can also pass a prompt to `pipe` such as:
+
+```python 
 prompt = "A fantasy landscape, trending on artstation"
 
 images = pipe(prompt, init_image).images
```
Comment (patrickvonplaten) [inline docs/source/en/api/pipelines/stable_unclip.mdx:72]:
```suggestion
images = pipe(init_image).images
```

Comment (sayakpaul) [reply docs/source/en/api/pipelines/stable_unclip.mdx:72]: Here we are telling the readers that they can also pass a prompt. Passing a prompt is not necessary is something we covered in the earlier section. See here: https://github.com/huggingface/diffusers/pull/2815/files#diff-0bec8187fde802d6ffcdfbad61c1d944e0a29179e185a6214514fb08ced13725R63

Comment (patrickvonplaten) [reply docs/source/en/api/pipelines/stable_unclip.mdx:71]: You're right - thanks!

File: docs/source/en/api/pipelines/stable_unclip.mdx [hunk 3]
```diff
@@ -54,19 +50,25 @@ from io import BytesIO
 from diffusers import StableUnCLIPImg2ImgPipeline
 
 pipe = StableUnCLIPImg2ImgPipeline.from_pretrained(
-    "fusing/stable-unclip-2-1-l-img2img", torch_dtype=torch.float16
-)  # TODO update model path
+    "stabilityai/stable-diffusion-2-1-unclip", torch_dtype=torch.float16, variation="fp16"
+)
 pipe = pipe.to("cuda")
 
 url = "https://raw.githubusercontent.com/CompVis/stable-diffusion/main/assets/stable-samples/img2img/sketch-mountains-input.jpg"
```
Comment (patrickvonplaten) [inline docs/source/en/api/pipelines/stable_unclip.mdx:57]:
```suggestion
url = "https://huggingface.co/datasets/hf-internal-testing/diffusers-images/resolve/main/stable_unclip/tarsila_do_amaral.png"
```

**Pull request merged**